schemas/common_definitions.json

```diff
@@ -42,12 +43,25 @@
         "type": "string",
         "description": "The text that is used for the 'Hide guidance' link."
       },
-      "content": {
-        "$ref": "common_definitions.json#/content"
+      "contents": {
+        "$ref": "common_definitions.json#/contents"
       }
     }
   },
   "content": {
+    "description": "Allows customisation of guidance text.",
+    "type": "object",
+    "properties": {
+      "title": {
+        "type": "string",
+        "description": "The H1 title on the page."
+      },
+      "contents": {
+        "$ref": "common_definitions.json#/contents"
+      }
+    }
+  },
+  "contents": {
```

Comment:
The description here is wrong, `contents` is used anywhere we need flexible content (headings, paragraphs, lists).